### PR TITLE
Add acrichto as a speaker

### PR DESCRIPTION
**Pull request by alexcrichton**
If this isn't quite right please just let me know!

### CODE REVIEW

File: _sessions/acrichto.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+permalink: /sessions/:name
+type: talk
+
+by: acrichto
+speaker: Alex Crichton
+title:
+desc: |
+  Async I/O has forever been a hot topic of discussion in Rust, but over the
+  past year we've seen some significant advances in this domain with the
+  `funtures` crate an the Tokio project. We'll start off by taking a look at
```
Comment (Hoverbear) [inline _sessions/acrichto.html]: I strongly approve of `funtures`.

File: _sessions/acrichto.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+layout: session
+permalink: /sessions/:name
+type: talk
+
+by: acrichto
+speaker: Alex Crichton
+title:
```
Comment (flaki) [inline _sessions/acrichto.html]: I'm just realizing we're missing the title here. 😶

Comment (flaki) [reply _sessions/acrichto.html]: @alexcrichton if you, by any chance, could fill this in, I would announce this around 8AM CEST

Comment (alexcrichton) [reply _sessions/acrichto.html]: Oops sorry! Done now

Comment (flaki) [reply _sessions/acrichto.html]: Thanks a bunch! This will go live in an hour. ;)

**Pull request merged**